content/en/docs/concepts/cluster-administration/node-autoscaling.md

```diff
@@ -0,0 +1,276 @@
+---
+reviewers:
+- gjtempleton
+- jonathan-innis
+- maciekpytel
+title: Node Autoscaling
+linkTitle: Node Autoscaling
+description: >-
+  Automatically provision and consolidate the Nodes in your cluster to adapt to demand and optimize cost.
+content_type: concept
+weight: 130
+---
+
+In order to run workloads in your cluster, you need
+{{< glossary_tooltip text="Nodes" term_id="node" >}}. Nodes in your cluster can be _autoscaled_ -
+dynamically [_provisioned_](#provisioning), or [_consolidated_](#consolidation) to provide needed
+capacity while optimizing cost. Autoscaling is performed by Node [_autoscalers_](#autoscalers).
+
+## Node provisioning {#provisioning}
+
+If there are Pods in a cluster that can't be scheduled on existing Nodes, new Nodes can be
+automatically added to the cluster - _provisioned_ - to accommodate the Pods. This is especially
+useful if the number of Pods changes over time, for example as a result
+of [combining workload and Node autoscaling](#multi-scaling).
+
+Autoscalers provision the Nodes by creating and deleting cloud provider resources backing them. Most
+commonly, the resources backing the Nodes are Virtual Machines (VMs).
+
+The goal of provisioning is to make all Pods schedulable. This goal is not always attainable because
+of various limitations (for example reaching configured provisioning limits, provisioning
+configuration not being compatible with a particular set of pods, lack of cloud provider capacity).
+
+{{< note >}}
+The scheduling constraints of pods (for example resource requests, node selectors) are the main
+input when determining Nodes to provision. In particular, the real resource usage of the Pods after
+they start running is not taken into account. In order to autoscale Nodes based on actual resource
+usage, you can [combine workload and Node autoscaling](#multi-scaling).
+{{< /note >}}
```

Comment:
There are a lot of notes.

Please drop the `note` shortcode, and if you like the rewording, you could try this:
```suggestion
The [scheduling constraints](/docs/concepts/scheduling-eviction/assign-pod-node/) of pods
(for example resource requests, and node selectors) are the main input to a node autoscaler 
when determining whether to provision a new server as a node, and if so what kind of server.
In particular, autoscalers don't directly take into account the real resource usage of the Pods
after they start running. In order to optimize your cluster and its nodes based on actual resource
usage, you can [combine workload and node autoscaling](#multi-scaling).
```